### ISSUE TITLE
Show entire content text if short enough

### ISSUE THREAD

**Issue by dshanske** [pfefferle/wordpress-webmention#284]
Related: https://github.com/pfefferle/wordpress-semantic-linkbacks/issues/41

"If the mention post is a note and/or its content text is short enough - say, < 140 chars :P - it'd be nice to show the content text inline."

**Comment by dshanske** [comment]
Would suggest we consider this for #361 but otherwise, it seems like it might conflict with the other presentation.

**Comment by pfefferle** [comment]
I think this issue is about the facepiles setting from SL, where you where able to decide if a type shows up as comment or as facepiles.

**Comment by dshanske** [comment]
I think this is specifically for type mention though.

**Comment by pfefferle** [comment]
Ah, this is a ticket created by you! There is another ticket about the setting! Sorry, forget what I commented ☺️

**Comment by dshanske** [comment]
@pfefferle Actually, it was bringing over one created by @snarfed

**Comment by pfefferle** [comment]
I see!

**Comment by pfefferle** [comment]
I think for this one we should change the comment type to "comment" if the requirements are met, to not make the output logic more complicated?

**Comment by snarfed** [comment]
Also related: https://github.com/pfefferle/wordpress-webmention/issues/182

**Comment by edent** [comment]
At the moment, I get around this by doing:

```php
$text_length = mb_strlen( wp_strip_all_tags( html_entity_decode( $comment_text, ENT_QUOTES ) ) );

if ( $text_length >= 1024 ) {
	$comment_text = "<a href=\"$comment_url\">Read more</a>.";
} else {
	$comment_text = apply_filters( 'comment_text', get_comment_text(), $comment);
}
```

But that could also truncate the text if it was too long.

**Comment by chrisbergr** [comment]
Maybe there should be an option to set from which character length the text is truncated? Or at least a filter.

**Comment by dshanske** [comment]
Resolved via #378 by having them set as comments.
